server/fixity.go

```diff
@@ -73,7 +73,7 @@ func (s *RESTServer) fixity() {
 	log.Println("Starting fixity loop")
 	for {
 		id := s.Fixity.NextFixity(time.Now())
-		if id == "" {
+		if id == "" || s.StatusTapeUse() == false {
```

Comment:
OK
---------- Forwarded message ----------
From: Don Brower <notifications@github.com>
Date: Thu, Aug 25, 2016 at 12:27 PM
Subject: Re: [ndlib/bendo] Issue 135 (#136)
To: ndlib/bendo <bendo@noreply.github.com>
Cc: Mark Suhovecky <suhovecky@nd.edu>, Author <author@noreply.github.com>


In server/fixity.go
<https://github.com/ndlib/bendo/pull/136#discussion_r76277368>:

> @@ -73,7 +73,7 @@ func (s *RESTServer) fixity() {
>  	log.Println("Starting fixity loop")
>  	for {
>  		id := s.Fixity.NextFixity(time.Now())
> -		if id == "" {
> +		if id == "" || s.StatusTapeUse() == false {

Don't call the function; access the constant directly.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/ndlib/bendo/pull/136/files/116b45889c785d5598950eba80d6730d33f98945#r76277368>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/ADNGTBYaVhIZxUgPjwUIv9zvGAEJKBNWks5qjcJygaJpZM4JsckT>
.